Examples/OneSignalDemo/app/src/main/AndroidManifest.xml

```diff
@@ -18,11 +18,11 @@
         android:theme="@style/AppTheme"
         tools:ignore="GoogleAppIndexingWarning">
 
-        <meta-data android:name="com.onesignal.NotificationExtensionServiceClass"
-            android:value="com.onesignal.sdktest.notification.AppNotificationExtensionService" />
+        <meta-data android:name="com.onesignal.NotificationServiceExtension"
+            android:value="com.onesignal.sdktest.notification.AppRemoteNotificationExtensionService" />
 
         <service
```

Comment:
This `service` should be remove. The NSE is no longer a service itself so it isn't needed. Only the `meta-data` tag is needed now.